src/components/Circles/EmptyComments.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+import { MessageSquare } from '@lifeomic/chromicons-native';
+import { useStyles } from '../../hooks';
+import { createStyles } from '../BrandConfigProvider';
+
+type Props = { style?: CirclesEmptyCommentStyles };
+
+export const EmptyComments = ({ style }: Props) => {
+  const { styles } = useStyles(defaultStyles, style);
+
+  return (
+    <View style={styles.container}>
+      <MessageSquare
+        stroke={styles?.iconTintColorImage?.tintColor}
+        style={styles.iconImage}
+      />
+
+      <Text style={styles.bodyText}>No comments yet.</Text>
+
+      <Text style={styles.helpText}>Be the first!</Text>
```

Comment:
Any reason not to put these in i18next from the get-go?

Reply:
All of the i18n stuff was just laziness on my part. I'll address these right away.